dht/token.cc

```diff
@@ -288,4 +288,14 @@ compaction_group_of(unsigned most_significant_bits, const token& t) {
     __builtin_unreachable();
 }
 
+token last_token_of_compaction_group(unsigned most_significant_bits, size_t group) {
```

Comment:
Are compaction groups groups really the target here?

Tablets should work without compaction groups, at least logically.

Reply:
Note that this is in `dht/token.cc`. This function is really about token space splitting, not really about compaction groups. Maybe we should rename those functions.

Reply:
I added this to mirror `compaction_group_of()`, which is defined right above it. That's what I was referring to. I also use it, and they are tightly related.

Reply:
Ok. It's an historical accident (compaction groups being introduced sooner). Right now it's possible for tablets and compaction groups to be split differently, but we can tighten it later and the distinction will disappear.